basket/news/tests/test_views.py

```diff
@@ -322,6 +324,15 @@ def _patch_views(self, name):
         self.addCleanup(patcher.stop)
 
 
+def mock_metrics(f):
+    @functools.wraps(f)
+    def wrapper(self, *args, **kwargs):
+        with MetricsMock() as mm:
+            return f(self, mm, *args, **kwargs)
```

Comment:
Because of the way you did this, the first arg is always the metrics mock. If you had other decorators adding arguments, the ordering of arguments becomes difficult to reason about. For that reason, I pass things in as keyword arguments instead.